### PR TITLE
DESCW-534 Change requests and accessibility features

### PR DESCRIPTION
**Pull request by codewisenate**
Add ARIA labels into queried content. Removes count numbers on accordion.

### CODE REVIEW

File: dist/public.asset.php [hunk 1]
```diff
@@ -1 +1 @@
-<?php return array('dependencies' => array(), 'version' => '24244cfb823cfde2c02a');
+<?php return array('dependencies' => array(), 'version' => 'c6c49a63c07f626533d5');
```
Comment (ShawnTurple) [inline dist/public.asset.php:1]: just wondering if you are using these files, if not could be something we add to .gitignore. There intention is to determine what dependencies are required and auto load, but seems like the only thing they are doing is creating an extra file in every commit.

File: inc/core/acf_blocks/sites/cleanbc/project-query.php [hunk 2]
```diff
@@ -88,7 +90,38 @@
 
                 <h4><?php echo esc_html( get_the_title( $project->ID ) ); ?></h4>
 
-                <?php echo wp_kses_post( get_the_content( null, false, $project->ID ) ); ?>
+                <?php echo wp_kses_post( get_the_content( null, true, $project->ID ) ); ?>
+                
+                <?php
+                $dom1 = new \DOMDocument();
```
Comment (ShawnTurple) [inline inc/core/acf_blocks/sites/cleanbc/project-query.php:100]: yuk!!! php dom stuff. is this the best way of doing this? Maybe a comment of why you are using php dom, as no one in my mind should be using this awful thing.  Could Javascript serve the same purpose?

**Review by ShawnTurple — APPROVED** [review]

File: inc/core/acf_blocks/sites/cleanbc/project-query.php (hunk 2 above)
Comment (codewisenate) [reply inc/core/acf_blocks/sites/cleanbc/project-query.php:100]: I don't disagree. This was the fastest solution I could come up with to grab the Wordpress meta comments associated with blocks. I added an aria label to buttons which works fine when the block is rendered directly but when it is queried as part of get_the_content it does not. It does dump the WP meta comment along side the base button block so this code grabs that and generates a div with a data attribute that can be used to inject an aria label into the button. It's not elegant and given time I will see if I can come up with a pure JS node traversal solution.

Comment (codewisenate) [reply inc/core/acf_blocks/sites/cleanbc/project-query.php:100]: Yeah I can do something with document.createNodeIterator, so I'll work on that next week as an improvement and consolidate it into the accessibility script.

**Pull request merged**